parlai/core/build_data.py

```diff
@@ -24,24 +24,84 @@
 from multiprocessing import Pool
 
 
-def download_check(dpath, urls, file_names, sha256, from_google=False):
-    # Download the data.
-    for i, url in enumerate(urls):
-        fname = file_names[i]
-        if from_google:
-            download_from_google_drive(url, os.path.join(dpath, fname))
+class DownloadableFile:
```

Comment:
😍

Reply:
I'm running the final test right now. I'll update the log as soon as it finishes!

Reply:
The CircleCI tests are failing because the link for mutliwoz is undermaintence.
https://www.repository.cam.ac.uk/bitstream/handle/1810/294507/MULTIWOZ2.1.zip?sequence=1&isAllowed=y